src/modal.ts

```diff
@@ -266,24 +273,36 @@ export default class GlideCoreModal extends LitElement {
 
         <footer>
           <menu class="actions">
-            <li class="action">
+            <li aria-hidden=${!this.hasTertiarySlotContent} class="action">
```

Comment:
Conditionally show each item to screenreaders so they're not presented as empty menu items.